lib/evaluate/call.h

```diff
@@ -137,6 +137,11 @@ class ActualArgument {
 
 using ActualArguments = std::vector<std::optional<ActualArgument>>;
 
+using ActualArgumentRef = common::Reference<const ActualArgument>;
+inline bool operator<(ActualArgumentRef x, ActualArgumentRef y) {
+  return &*x < &*y;
+}
```

Comment:
for std::set<evaluate::ActualArgumentRef>? If there is only one use should we move this to the place where the set is defined?

Reply:
Good idea.  I had to flail a little to get the namespace for the operator correct, but I finally figured it out.